tests/integration/test_grpc_protocol_ssl/configs/grpc_config.xml

```diff
@@ -0,0 +1,24 @@
+<yandex>
+    <grpc_port>9001</grpc_port>
```

Comment:
Maybe change all ports to 9100?
Because 9001 is often used for second ClickHouse instance if multiple instances are run on the same server.

Reply:
I'm changing them in https://github.com/ClickHouse/ClickHouse/pull/17435